server_src/runners/softwareUpdateChecker.js

```diff
@@ -1,7 +1,8 @@
 const Logger = require("../lib/logger.js");
-const { getGithubReleasesPromise } = require("./githubClient");
+const {AppConstants} = require("../app.constants");
+const {getGithubReleasesPromise} = require("./githubClient");
 
-const logger = new Logger("OctoFarm-SoftwareUpdateChecker");
+const logger = new Logger("OF-SoftwareUpdateChecker");
```

Comment:
Minor here but I think "OctoFarm" was fine initially unless we're changing them all to match?